modules/core/src/v2/wallet.ts

```diff
@@ -2587,4 +2643,146 @@ export class Wallet {
       };
     }
   }
+
+  // #region TSS region
+
+  async sendManyTSS(params: TSSSendManyOptions): Promise<any> {
```

Comment:
Function is too long. We need verify here if we can reduce it. But on the other hand I know you need to do execute several calls for the shares to final execute the send many for tss.

Reply:
Its the pipeline design we talked about, its better than the current sendMany i guess.